src/senaite/core/browser/viewlets/templates/sampleheader.pt

```diff
@@ -1,7 +1,7 @@
 <div id="senaite-sampleheader"
      tal:define="senaite_view python:context.restrictedTraverse('@@senaite_view');
                  test nocall:senaite_view/test;
-                 errors python:request.get('errors', {});
+                 errors python:options.get('errors', {});
```

Comment:
huh, this is a hidden gem. I did not know about the built-in `options` variable and that it was possible to pass data to the template with keywords arguments (`self.template(errors=errors)` in this case)!

Reply:
Yes, this is quite helpful and used here as well for example:

https://github.com/senaite/senaite.impress/blob/2.x/src/senaite/impress/templates/reports/MultiDefault.pt#L8

https://github.com/senaite/senaite.impress/blob/2.x/src/senaite/impress/ajax.py#L187

https://github.com/senaite/senaite.impress/blob/257a09291aa2ea087623b18dcff687c604c61e2d/src/senaite/impress/publishview.py#L342